core/src/main/resources/error/error-classes.json

```diff
@@ -1046,6 +1046,63 @@
       "Protobuf type not yet supported: <protobufType>."
     ]
   },
+  "PYSPARK" : {
+    "message" : [
+      ""
+    ],
+    "subClass" : {
+      "COLUMN_IN_LIST" : {
+        "message" : [
+          "<funcName> does not allow a column in a list"
+        ]
+      },
+      "HIGHER_ORDER_FUNCTION_SHOULD_RETURN_COLUMN" : {
+        "message" : [
+          "Function '<funcName>' should return Column, got <returnType>"
```

Comment:
`'<funcName>'` should be quoted by backticks.

Reply:
Thanks for the comments, @MaxGekk . Let me address it soon.